XML3-Homework/Giles_Jan_26_Exercise_3.xml

```diff
@@ -0,0 +1,145 @@
+            <location>Adventurer's Guild</location> and talking to <character>Katheryne</character>
+        of <region>Inazuma</region>. </p>
+    <subheader><character>Arataki Itto</character> Story Quest</subheader>
+
+    <p>Taurus Iracundus Chapter</p>
+
+    <p step="1">Talk to <character>Katheryne</character> in <region>Inazuma</region></p>
+
+    <p step="2">Talk to <character>Furuya Noboru</character></p>
+
+    <p step="3">Talk to <character>Ogura Mio</character></p>
+
+    <p step="4">Talk to <character>Uesugi</character></p>
+
+    <p step="5">Talk to <character>Sango</character> Obtain: <questItem>A Bag of
+        Beans</questItem></p>
+
+    <p step="6">Go to <location>Yashiori Island</location> and search for <character>Arataki
+            Itto</character> To Do: Use Elemental Sight and follow the foot trail</p>
+
+    <p step="7">Throw beans at <character>Arataki Itto</character> Note: Avoid getting caught by
+        him</p>
+
+    <p step="8">Go to the <location>Onikabuto capture shot</location> (above the hill)</p>
+
+    <p step="9">Go to <location>a certain tree around the area</location> and catch an Onikabuto
+        with Itto</p>
+
+    <p step="10">Go down to <location>the cliff</location> and go to <location>the next
+            tree</location> to search for another Onikabuto</p>
+
+    <p step="11">Go to a <location>certain rock</location>, where you will find (1) one
+        Onikabuto</p>
+
+    <p step="12">Go back <location>near the camp</location> and have a Bettle battle with
+            <character>Daisuke</character> and <character>Itto</character></p>
+
+    <p step="13">Go to <location>Fort Fujitou</location></p>
+
+    <p step="14">Talk to <character>Masato</character></p>
+
+    <p step="15">Go to <location>Nazuchi Beach</location></p>
+
+    <p step="16">Defeat the enemies Character Trial: <character>Arataki Itto</character></p>
+
+    <p step="17">Defeat the Tenryou Guards Character Trial: Arataki Itto</p>
+
+    <p step="18">Follow the footprints in order to chase <character alias="blueOni"
+            >Takuya</character> (Blue Oni)</p>
+
+    <p step="19">Search for a <location>Hidden Hideout</location>.</p>
+
+    <p step="20">Enter the domain and defeat the enemies Event Domain: Crumbling Guilt Character
+        Trial: <character>Arataki Itto</character></p>
+
+    <p step="21">Go back to <character>Grandma Oni</character></p>
+
+    <subHeader>Obtain A <questItem>Bag Of Beans</questItem></subHeader>
+
+    <p>After talking to <character>Sango</character>, she will give you A <questItem>Bag of
+            Beans</questItem> that will be used in order to capture <character>Arataki
+            Itto</character>. This item is only available on <character>Arataki Itto's</character>
+        quest and its usage is still unknown.</p>
+
+    <header><character>Arataki Itto</character> Story Quest - Rewards</header>
+
+    <subHeader>Rewards List</subHeader>
+
+    <li>
```

Comment:
@julianjmg This `<li>` seems to be borrowed from HTML, which is okay, but maybe not as descriptive as it could be. Maybe name it `<reward>`? Think of the semantic markup you do for XML as the thing makes it yours and makes it very clear what information we find.